src/__tests__/createStore.test.js

```diff
@@ -13,7 +13,23 @@ const storeWithIgnore = storeFactory({ ignore: ['bar', 'baz'] })
 const storeWithOnly = storeFactory({ only: ['foo'] })
 const storeWithConflicts = storeFactory({ only: ['bar'], ignore: ['bar'] })
 
-beforeEach(() => localStorage.clear())
+const errorHandler = jest.fn()
+const errorStoreAdapter = {
+  get(key) { throw Error('Error getting key') },
+  set(key, value) { throw Error('Error setting key') },
+  clear() { throw Error('Error clearing storage') },
+  remove(key) { throw Error('Error removing key') },
+  onGetError(err, key) { errorHandler(err, key) },
+  onSetError(err, key, data) { errorHandler(err, key, data) },
```

Comment:
For consistency with the `set` signature.

```suggestion
  onSetError(err, key, data) { errorHandler(err, key, value) },
```

Reply:
```suggestion
  onSetError(err, key, value) { errorHandler(err, key, value) },
```